File: codewars/8kyu/remove-first-and-last-character.py
# SOLUTION
# ==============================================================================
def remove_char(s):
    return s[1:-1]

# TESTS
# ==============================================================================
if __name__ == "__main__":
    assert remove_char("hello") == "ell", "Test case 1 failed"
    assert remove_char("world") == "orl", "Test case 2 failed"
    assert remove_char("a") == "", "Test case 3 failed"
    assert remove_char("ab") == "", "Test case 4 failed"
    assert remove_char("12345") == "234", "Test case 5 failed"
    print("All test cases passed!")

File: codewars/8kyu/remove-first-and-last-character.cpp
#include <iostream>
#include <string>
#include <cassert>

std::string sliceString(std::string str) {
    return str.substr(1, str.length() - 2);
}

void runTests() {
    assert(sliceString("tuna") == "un");
    assert(sliceString("rats") == "at");
    assert(sliceString("code") == "od");
    assert(sliceString("ok").empty());
    std::cout << "All tests passed!" << '\n';
}

int main() {
    runTests();
    return 0;
}
